MathLib/LinAlg/Lis/LisLinearSolver.cpp

```diff
@@ -70,62 +42,64 @@ void LisLinearSolver::solve(LisVector &b, LisVector &x)
 
     INFO("------------------------------------------------------------------");
     INFO("*** LIS solver computation");
-#ifdef _OPENMP
```

Comment:
Looks great. Only one question hereby: where to print number of threads used by LIS if this ifdef is removed?